player/external_files.c

```diff
@@ -43,32 +43,48 @@ static const char *const audio_exts[] = {"mp3", "aac", "mka", "dts", "flac",
                                          NULL};
 
 static const char *const image_exts[] = {"jpg", "jpeg", "png", "gif", "bmp",
-                                         "webp", "jxl", "tiff", "tif",
+                                         "webp", "jxl", "tiff", "tif", "avif",
                                          NULL};
 
 // Stolen from: vlc/-/blob/master/modules/meta_engine/folder.c#L40
 // sorted by priority (descending)
 static const char *const cover_files[] = {
```

Comment:
wouldn't be better to prefer higher quality formats first?

Reply:
@kasper93, I think I may prefer high quality formats quite yet.
JPEG XL is lossless and with JXL lossy is better score result than AVIF lossy.

- Martin Eesmaa